src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -5821,9 +5821,12 @@ void Test2()
 ", parseOptions: TestOptions.Regular8);
 
             c.VerifyDiagnostics(
-                 // (12,18): warning CS8602: Possible dereference of a null reference.
-                 //         var z1 = u1[0];
-                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "u1").WithLocation(12, 18)
+                // (11,14): warning CS8600: Cannot convert null to non-nullable reference.
```

Comment:
>// (11,14): warning CS8600: Cannot convert null to non-nullable reference. [](start = 16, length = 74)

I think the purpose of this test was to ensure that appropriate warnings are produced for ```u1[0]``` and none produced for ```u1?[u1[0]]``` when ```u1``` might be null. Do we still have coverage for scenarios like that in other unit-tests?

Reply:
Updated test.